way_to_home/custom_user/views.py

```diff
@@ -116,3 +125,68 @@ def signin_google(request):
         return HttpResponse("User was successfully created", status=201)
 
     return HttpResponse("User's data is empty", status=400)
+
+
+@require_http_methods(["POST"])
+def reset_password(request):
+    """Function that provides reset user password"""
+    data = request.body
+    email = data.get('email')
+    if not email:
+        return RESPONSE_400_OBJECT_NOT_FOUND
```

Comment:
You need to use here `email_validator` and return `invalid_data` if `email` is invalid.
```suggestion
    if not email:
        return RESPONSE_400_OBJECT_NOT_FOUND
```

Reply:
done